src/main/java/fun/epak/pak/service/UserService.java

```diff
@@ -35,8 +40,25 @@ public User loadUserByEmail(String email) throws UsernameNotFoundException {
     }
 
     public void saveUser(User user) {
-        user.setPassword(passwordEncoder.encode(user.getPassword()));
         userRepository.save(user);
     }
 
+    public void deleteUser(Long id){
+        userRepository.deleteById(id);
+    }
+
+    public void registerUser(UserRegistrationRequest userRegistrationRequest){
+        User user = User.builder()
+                .email(userRegistrationRequest.getEmail())
+                .password(passwordEncoder.encode(userRegistrationRequest.getPassword()))
+                .username(userRegistrationRequest.getUsername())
+                .imageAddress(userRegistrationRequest.getImageAddress())
+                .isActive(true)
+                .registerDate(LocalDate.now())
+                .build();
+        saveUser(user);
```

Comment:
Nie potrzebujemy raczej osobnej metody, żeby wywołać save na repo :)

Reply:
Zostawiłem save w repo, bo jak ktoś będzie chciał zmienić dane, to wykorzystam tę metodę